x/programs/cmd/simulator/src/lib.rs

```diff
@@ -3,7 +3,8 @@
 //! Alternatively the `Plan` can be written in JSON and passed to the
 //! Simulator binary directly.
 
-use serde::{Deserialize, Serialize};
+use base64::{engine::general_purpose::STANDARD as b64, Engine};
+use serde::{ser::SerializeStruct, Deserialize, Serialize};
 use serde_with::{serde_as, DisplayFromStr};
```

Comment:
The `serde_with` dependency will be dropped once we merge https://github.com/ava-labs/hypersdk/pull/935

Reply:
edit: probably after manually implementing `Serialize` on the steps once we get rid of plans.